department-of-veterans-affairs/caseflow
# Non-modal form for converting travel board request type to virtual

Resolves #15157 

### Description
- serialize regional office label
- create new component `HearingTypeConversionForm` to display form to convert hearing type to virtual
- write stories for `HearingTypeConversionForm`
- rename method `getAppellantTitleForHearing` to the more general `getAppellantTitle`
- minor changes to  components `AppellantSection` and `VirtualHearingSecion`
- write jest tests for `HearingTypeConversionForm`

### Testing Plan
1. run `make storybook`
2. go to storybook and see `HearingConversionTypeForm` 

### Storybook Story
*For Frontend (Presentationa) Components*
